packages/sysutils/busybox/scripts/init

```diff
@@ -81,9 +82,11 @@
     case $arg in
       BOOT_IMAGE=*)
         IMAGE_KERNEL="${arg#*=}"
+        IMAGE_KERNEL=$(basename $IMAGE_KERNEL)
```

Comment:
Since `IMAGE_KERNEL` is used elsewhere (ie. [here](https://github.com/vpeter4/LibreELEC.tv/blob/img_update/packages/sysutils/busybox/scripts/init#L689), changing this value could have unintended consequences.

I would suggest using a different variable, or better yet simply apply the `basename()` call when you need the basename version of IMAGE_KERNEL, ie. in lin 643: `UPDATE_KERNEL="$(basename $IMAGE_KERNEL)"`